charts/canton-domain/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 apiVersion: v2
 name: "canton-domain"
-version: 0.0.6
+version: 0.1.0
```

Comment:
Let's bump the chart version in the private repository when we are ready, this will unsync things

Reply:
PR for that is here: https://github.com/DACH-NY/daml-enterprise-k8s-support/pull/96/files